modes/ripgrep/evil-collection-ripgrep.el

```diff
@@ -0,0 +1,50 @@
+;;; evil-collection-ripgrep.el --- Bindings for `ripgrep' -*- lexical-binding: t -*-
+
+;; Copyright (C) 2018 James Nguyen
```

Comment:
Set the year and the name to yours ;)

Reply:
I actually don't claim copyright on [any of my work](https://github.com/countvajhula/symex.el/blob/6816417e71c4cb28f30c6d7c9917ca71f3d6550f/symex.el#L9-L23) and would prefer not to. Is it OK if I just leave the line as is and update the year? Another possibility is to eliminate the line entirely, as I do in my Emacs extensions.

Reply:
As far as I understand it, the copyright line must appear for GPL work.  I'm not sure how it works if you want to surrender the rights to someone else, maybe it's not as straightforward as you did it here.  With the Free Software Foundation there is a piece of paper to sign, don't know if it would be the same here.

GPL is different from public domain in that it takes a political stance against proprietary licensing.

What this copyright line effectively does, if I understand correctly, is grant you the right to change the license to this piece of code, that's it.

Reply:
Nowadays we no longer need to put an explicit copyright line for copyright to still apply (e.g. [this link](https://www.newmediarights.org/business_models/artist/do_i_need_put_copyright_notice_my_work#:~:text=No.,longer%20necessary%20to%20receive%20protection.) talks about this). It's taken as a given from a legal perspective that at a minimum, copyright always applies for any creative work (but not patents etc). [This link](https://www.copyright.gov/help/faq/faq-general.html) also makes that explicit.

So the main reason to have a copyright line seems to be to simply record the names of the authors and the year of authorship as it relates to copyright. But that is already accomplished with version control much more reliably, and with [cryptographically secure hashes](https://gist.github.com/masak/2415865) (well, almost) recording uniquely identifying information about each commit, so it's much more reliable than a manually maintained list of authors for every file (which is probably inaccurate at scale anyway). This line may at this point just be a relic from traditional media. WDYT?

Reply:
Nowadays we no longer need to put an explicit copyright line for copyright to still apply (e.g. [this link](https://www.newmediarights.org/business_models/artist/do_i_need_put_copyright_notice_my_work#:~:text=No.,longer%20necessary%20to%20receive%20protection.) talks about this). It's taken as a given from a legal perspective that at a minimum, copyright always applies for any creative work (but not patents etc). [This link](https://www.copyright.gov/help/faq/faq-general.html) also makes that explicit.

So the main reason to have a copyright line seems to be to simply record the names of the authors and the year of authorship as it relates to copyright. But that is already accomplished with version control much more reliably, and with [cryptographically secure hashes](https://gist.github.com/masak/2415865) (well, almost) recording uniquely identifying information about each commit, so it's much more reliable than a manually maintained list of authors for every file (which is probably inaccurate at scale anyway). This line may at this point just be a relic from traditional media.